presidio-anonymizer/presidio_anonymizer/entities/anonymizer_request.py

```diff
@@ -26,23 +24,25 @@ def __init__(self,
         self._transformations = {}
         self._analysis_results = AnalyzerResults()
         self.__validate_and_insert_input(data)
-        self.default_transformation = {"type": "replace",
-                                       "anonymizer": self.anonymizers["replace"]}
+        self.default_transformation = {
+            "type": "replace",
+            "anonymizer": self.anonymizers["replace"],
+        }
 
-    def get_transformation(self, analyzer_result: AnalyzerResult):
+    def get_transformation(self, entity_type: str):
```

Comment:
nit: This could be private as it's only being called from private methods (if we stay with the __get_transformation_from_analyzer_result method)

Reply:
I've reverted and removed the __get_transformation_from_analyzer_result  method, so leaving it public